.github/workflows/update-gemini-default.yml

```
@@ -0,0 +1,50 @@
+
+      - name: Update Google default model and rebuild clean commit
+        shell: bash
+        run: |
+          git fetch origin master
+          python3 <<'PY'
+          from pathlib import Path
+
+          path = Path('src/config/index.mjs')
+          text = path.read_text(encoding='utf-8')
+          old = """  'googleGemini3_1Pro',
+  'googleGemini3_5Flash',
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)